PensionManagementPensionerService/Controllers/GuardianController.cs

```diff
@@ -65,9 +65,8 @@ public async Task<IActionResult> UpdateGuardianById(Guid guardianId, [FromBody]
         {
             try
             {
-                var result = await _guardianRepository.GetGuardianById(guardianId);
                 await _guardianRepository.UpdateGuardianById(guardianId, guardianDetails);
-                return Ok(result);
+                return Ok(guardianDetails);
```

Comment:
is your update method not returning the output?

Reply:
it is returning the guardianDetails which we updated.